package/skelcd-control-openSUSE.spec

```diff
@@ -137,9 +137,9 @@ install -m 644 control/${CONTROL_FILE} $RPM_BUILD_ROOT%{?skelcdpath}/CD1/control
     # Leap debug repo (from :Update) has a different path since all 'ports' (ARM and PPC) are in the same repo
     sed -i -e "s,http://download.opensuse.org/debug/update,http://download.opensuse.org/ports/debug/update," %{buildroot}%{?skelcdpath}/CD1/control.xml
     sed -i -e "s,http://download.opensuse.org/debug/,http://download.opensuse.org/ports/$ports_arch/debug/," %{buildroot}%{?skelcdpath}/CD1/control.xml
-    sed -i -e "s,http://download.opensuse.org/source/,http://download.opensuse.org/ports/$ports_arch/source/," %{buildroot}%{?skelcdpath}/CD1/control.xml
     sed -i -e "s,http://download.opensuse.org/update/leap/,http://download.opensuse.org/ports/update/leap/," %{buildroot}%{?skelcdpath}/CD1/control.xml
     sed -i -e "s,http://download.opensuse.org/update/tumbleweed/,http://download.opensuse.org/ports/$ports_arch/update/tumbleweed/," %{buildroot}%{?skelcdpath}/CD1/control.xml
+    # For source-repo keep same URI for all arches.
```

Comment:
is same true also for aarch? is really all sources shared and ports does not contain specific patches?

Reply:
as far I understand OBS related projects, only the build related packages are specific, all other packages are in "Inherited Packages"
 https://build.opensuse.org/project/show/openSUSE:Leap:15.1:PowerPC
 https://build.opensuse.org/project/show/openSUSE:Leap:15.1:ARM

@ggardet could confirm.

Reply:
For Leap, we should have the same sources indeed. (Not true for Tumbleweed since release cadence is different). But I would be more in favor to have redirect rules on download server, especially since 15.1 has been released with those repo URLs.